index.html

```diff
@@ -72,8 +72,200 @@ <h1 class="intro">
+          alt="project card display"
+        />
+        <div class="description-container">
+          <h1>Men's Fashion</h1>
+          <p class="description">
+            A clothes advertisment page for classic men's wear
+          </p>
+          <ul class="pale-info-container">
+            <li class="pale-info">html</li>
+            <li class="pale-info">Css</li>
+            <li class="pale-info">JavaScript</li>
+          </ul>
+          <div class="action">
+            <button class="see-project" type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <!-- card no. 3 -->
+      <div class="work-item">
+        <img
+          class="work-card"
+          src="./Assets/school_img1.PNG"
+          alt="project card display"
+        />
+        <div class="description-container">
+          <h1>JOEJOY INC.</h1>
+          <p class="description">A sample tertiary level school website.</p>
+          <ul class="pale-info-container">
+            <li class="pale-info">html</li>
+            <li class="pale-info">Css</li>
+            <li class="pale-info">JavaScript</li>
+          </ul>
+          <div class="action">
+            <button class="see-project" type="button">Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- About section -->
+    <section class="about-section">
+      <div class="top">
+        <h1 class="intro">About Myself</h1>
+
+        <p class="description-container">
+          I'm a software developer! I can help you build a product , feature or
+          website. Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
+        <span class="connect">LET'S CONNECT</span>
+        <div class="social">
+          <ul>
+            <li>
+              <a href="#"><span class="fa fa-twitter twitter"></span></a>
+            </li>
+            <li>
+              <a href="#"><span class="fa fa-linkedin"></span></a>
+            </li>
+            <li>
+              <a href="#"><span class="fa fa-medium"></span></a>
+            </li>
+            <li>
+              <a href="#"><span class="fa fa-github"></span></a>
+            </li>
+            <li>
+              <a href="#"><span class="fa fa-hand-peace"></span></a>
+            </li>
+          </ul>
+        </div>
+        <div class="for-space"></div>
+      </div>
+      <div class="action">
+        <button class="see-project" type="button">Get My Resume</button>
+      </div>
+
+      <!-- For skills section -->
+      <section>
+        <div class="main-list">
+          <!-- skill item 1 -->
+          <div class="skill-item">
+            <div class="skill-headline">
+              <div>
+                <h3>Languages</h3>
+              </div>
+              <div>
+                <a href="#"
+                  ><img
+                    class="icon"
```

Comment:
- [ ] The width of the image should be 100% width of its parent. thats why the image is not fully filled. Kindly fix all th image in work-section.
- Your Design
![your](https://user-images.githubusercontent.com/26767383/148652260-25a0f24c-3c5c-4d7f-af45-cd6191ba68c9.PNG)
- Original design
![edit](https://user-images.githubusercontent.com/26767383/148652274-de0f64cd-57a6-444b-9f6c-6386e15a2f35.PNG)